test/pilosa.go

```diff
@@ -380,6 +425,17 @@ func MustDo(method, urlStr string, body string) *httpResponse {
 	return &httpResponse{Response: resp, Body: string(buf)}
 }
 
+func CheckGroupBy(t *testing.T, expected, results []pilosa.GroupCount) {
+	if len(results) != len(expected) {
+		t.Fatalf("number of  groupings mismatch:\n got:%+v\nwant:%+v\n", results, expected)
```

Comment:
I realize this doesn't matter, and was just copied code, but there is an extra space after "of"

Reply:
updated